test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Script.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@addtaghelper "*, Microsoft.AspNet.Mvc.TagHelpers"
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Script</title>
+    <script src="~/blank.js" data-foo="foo-data1">
+        // Regular script with comment in body, and extra properties.
+    </script>
+
+    <script src="~/blank.js" asp-fallback-src="~/fallback.js" asp-fallback-test="blank.wooptido()" data-foo="foo-data2">
```

Comment:
In order to not confuse users I don't think `asp-fallback-test="blank.wooptido()"` should be used here. Rarely will a user execute a method to test success of a script inclusion, should change this to just be `blank.wooptido` (aka, if the method exists then the script was included correctly).